retroarch/retroarch.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "retroarch",
+	"config-opts": [ "--enable-dbus" ],
+	"make-args": [ "GLOBAL_CONFIG_DIR=${FLATPAK_DEST}/etc" ],
+	"sources": [
+		{
+			"type": "git",
+			"url": "https://github.com/libretro/RetroArch.git",
+			"branch": "9b282aa742b6c3d2f2925ae5a12e2cd7c6b6ad38"
+		},
+		{
+			"type": "file",
+			"path": "org.libretro.RetroArch/retroarch.cfg"
+		}
+	],
+	"post-install": [
+		"mkdir -p ${FLATPAK_DEST}/etc",
+		"sed s:@prefix@:${FLATPAK_DEST}:g retroarch.cfg > ${FLATPAK_DEST}/etc/retroarch.cfg"
+	],
+	"modules": [
+		"org.libretro.RetroArch/libpng/libpng-1.6.35.json",
```

Comment:
what's wrong with the libpng in the runtime?

Reply:
As it turns out, nothing. I just copied this from what the RetroArch flatpak does, but it seems to be unnecessary. Will remove.